docs/user_manual/calculations.md

```diff
@@ -466,8 +516,38 @@ $$
     \end{eqnarray}
 $$
 
-- Branch/shunt power flow: Linear WLS requires a complex current phasor. To make this translation, the voltage at the terminal should
-also be measured, otherwise the nominal voltage with zero angle is used as an estimation. With the measured (linearized) voltage
+- Branch current with global angle: The global angle current measurement captures
+the phase offset relative to the same predetermined reference phase against which the voltage angle is measured. It is not
+sufficient to use the default zero-phase reference offset, because the reference point is not uniquely determined when there
+are no voltage angle measurements, resulting in ambiguities. As a result, using any global angle current sensor in the grid
+requires at least one voltage phasor measurement (with angle).
+
+$$
+    \begin{eqnarray}
+            \underline{I} = I_i \cdot e^{j \theta_i}
+    \end{eqnarray}
+$$
+
+- Branch current with local angle: Sometimes, (accurate) voltage measurements are available for a branch,
+which means no power measurement is possible. While it is not straightforward to use magnitude-only measurements of currents in
```

Comment:
`Sometimes, (accurate) voltage measurements are available for a branch,
which means no power measurement is possible. `
Not sure if I understand the intent here correctly.

Reply:
Ahh, maybe you meant
```suggestion
- Branch current with local angle: Sometimes, (accurate) voltage measurements are not available for a branch,
which means no power measurement is possible. While it is not straightforward to use magnitude-only measurements of currents in
```